docs/RC/GetInteriorVehicleDataConsent/index.md

```diff
@@ -26,14 +26,15 @@ The driver is expected to have an ability to grant or deny the permission.
 |Name|Type|Mandatory|Additional|Description|
 |:---|:---|:--------|:---------|:----------|
 |moduleType|[Common.ModuleType](../../common/enums/#moduletype)|true| |The module type that the app requests to control|
+|moduleIds|String|false|maxlength="100"|Ids of a module of same type, published by System Capability|
 |appID|Integer|true| |ID of the application that triggers the permission prompt|
 
 ### Response
 
 #### Parameters
 |Name|Type|Mandatory|Additional|Description|
 |:---|:---|:--------|:---------|:----------|
-|allowed|Boolean|true| |"true" - if the driver grants the permission for controlling to the named app; <br> "false" - in case the driver denies the permission for controlling to the named app.|
+|allowed|Boolean|true|array="true"|"true" - if the driver grants the permission for controlling to the named app; <br> "false" - in case the driver denies the permission for controlling to the named app.|
```

Comment:
```suggestion
|allowed|Boolean|true|array: true|"true" - if the driver grants the permission for controlling to the named app; <br> "false" - in case the driver denies the permission for controlling to the named app.|
```